web_client/templates/dialogs/saveAnnotation.pug

```diff
@@ -9,7 +12,7 @@
         .form-group
           label(for='h-annotation-name') Name
           input#h-annotation-name.input-sm.form-control(
-            type='text', placeholder='Enter a name for the annotation', value=annotation.name)
+            type='text', placeholder='', value=defaultName)
```

Comment:
If the annotation already has a name, this won't show it.  We probably want to leave the placeholder (since it will show if the user deletes the entire name), and have `value=annotation.name || defaultName`)

Reply:
This will likely fix the annotation test.

Reply:
I have made the changes and tested locally, it passed all the tests.
I just pushed again.